packages/manager/apps/pci-object-storage/public/translations/pci-object-storage/storages/s3/objects/Messages_fr_FR.json

```
@@ -60,5 +59,14 @@
   "extendRestoreObjectToastSuccessDescription": "La prolongation a bien été prise en compte",
   "inProgressRestore": "Restauration en cours...",
   "available": "Disponible jusqu'au",
-  "seeVersionsSwitchLabel": "Voir les versions"
+  "seeVersionsSwitchLabel": "Voir les versions",
+  "selectObject": "Sélectionner l'objet {{name}}",
+  "selectAll": "Tout sélectionner",
+  "bulkDeleteTitle": "Supprimer plusieurs objets",
+  "bulkDeleteDescription": "Vous allez supprimer {{count}} objet(s).",
+  "bulkDeleteButtonConfirm": "Supprimer {{count}} objet(s)",
+  "bulkDeleteToastSuccessDescription": "{{count}} objet(s) supprimé(s) avec succès.",
+  "bulkDeleteVersionsTitle": "Supprimer plusieurs versions",
+  "bulkDeleteVersionsDescription": "Voulez-vous supprimer définitivement {{count}} version(s)? Cette action ne peut être annulée.",
+  "bulkDeleteVersionsConfirmation": "Taper PERMANENTLY DELETE pour confirmer"
```

We use `-ez` form on other similar confirm messages

```suggestion
  "bulkDeleteVersionsConfirmation": "Tapez PERMANENTLY DELETE pour confirmer"
```